src/reference-manual/statements.qmd

```diff
@@ -435,20 +466,20 @@ and explicitly incrementing the log probability function, as in
 target += normal_lpdf(y | mu, sigma);
 ```
 
-The sampling statement drops all the terms in the log probability
+The distribution statement drops all the terms in the log probability
```

Comment:
This isn't your fault, but the section here should really mention you can use `normal_lupdf` and reproduce the distribution statement behavior.  I honestly don't understand why everyone's using lpdf in their code.

Reply:
As this is discussed later, I know added a sentence referring to a later discussion, but I can make a bigger rearrangement if you think that would be useful. I don't everyone uses, but a) I haven't seen much speed difference in most models, b) some people use it just in case that someone might want to use their model and compute Bayes factor (e.g. this is why brms uses them).

Reply:
OK, I think that's fine for the reference manual.  We should probably add more stuff to the User's Guide on this, but that's a separate PR.